region/maridia/inner-yellow/Plasma Spark Room.json

```diff
@@ -1189,6 +1189,30 @@
         "FIXME: A 75% damage reduction method needs to be documented for if Samus only has Gravity and it provides 75% reduction."
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
```

Comment:
Either pause abuse the Owtch, or write logic for killing the right side owtches and using a fish/choot.